Update rdpwrap.ini

shubell opened the pull request:
Compared to my rdpwrap.ini file, added a few comments and a few version. For some I am not sure which is the correct version.

asmtron commented:
Hi DRSDavidSoft,

[10.0.17763.165] the value of SingleUserOffset.x64=1322C is correct
0000000180013229 C7 40 08 01 00 00 00                          mov     dword ptr [rax+8], 1

[10.0.17763.437] the value of SingleUserOffset.x64=1339C is correct
0000000180013399 C7 40 08 01 00 00 00                          mov     dword ptr [rax+8], 1

[10.0.18362.1] the value of SingleUserOffset.x64=0DBFC is correct
000000018000DBF9 C7 40 08 01 00 00 00                          mov     dword ptr [rax+8], 1

DRSDavidSoft commented:
@asmtron Thank you for the correct values. It seems (for at least) `10.0.17763.437`, the [alternative one](https://github.com/stascorp/rdpwrap/pull/725/files#r292265309) also works.

In this branch, I have kept the values that you suggest are correct.

At this point, this PR only adds `10.0.18908.1000` in addition to some general clean up (see [here](https://github.com/asmtron/rdpwrap/pull/1/files)).

Would you please merge it so I can rebase my branch off of yours as upstream? :)
Thank you for maintaining it!

DRSDavidSoft commented on res/rdpwrap.ini:
```diff
@@ -3997,7 +4017,6 @@ bServerSku.x86        =C3F74
 ulMaxDebugSessions.x86=C3F78
 bRemoteConnAllowed.x86=C3F7C
 
-zlMaxUserSessions.x64  =F23B0
```
@asmtron `zlMaxUserSessions` is bogus and should be removed

asmtron commented:
new ini updates 2020

graphixillusion commented:
@asmtron Hi there. Now that the main repo is closed, what about to open issues section over here? Btw, thank you for the last update. Do you have fix for version 2004 too?

shippj commented:
thank you so much for the quick update @asmtron .  Since the main repo was closed "due to a violation of GitHub's terms of service", would you mind picking a new meeting place and posting it here just in case your repo gets closed too?  Maybe a decentralized system, such as mastodon?

shippj commented:
win10 32bit is still showing the "Another user is signed in. If you continue, they'll be disconnected" message.

![image](https://user-images.githubusercontent.com/21339238/82106587-ababdb80-96e7-11ea-9d39-1f07fbbabfc7.png)

asmtron commented:
 

PLease send me the termsrv.dll of your win10 32bit (C:\Windows\System32\termsrv.dll) [version 10.0.18362.836]
 

Sent: Saturday, May 16, 2020 at 1:09 AM
From: "shippj" <notifications@github.com>
To: "asmtron/rdpwrap" <rdpwrap@noreply.github.com>
Cc: "asmtron" <asmtron@mail.com>, "Mention" <mention@noreply.github.com>
Subject: Re: [asmtron/rdpwrap] Updates with new builds (#1)


 

win10 32bit is still showing the "Another user is signed in. If you continue, they'll be disconnected" message.



—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.

shippj commented:
here ya go, thanks!

[termsrv.zip](https://github.com/asmtron/rdpwrap/files/4640792/termsrv.zip)

shippj commented:
so, i'm trying to learn how to do this myself.  I used to play around with IDA back about 15-20 years ago but haven't had the need to mess with it since and i've forgotten most of what I knew.  I was delighted to see that IDA is now free for home use, and it works great on Linux, so I fired it up and found a "find THIS hex string and replace it with THIS hex string" tutorial for cracking the 18362.657 version of the DLL.  After learning how to find that hex string in IDA, I noticed the address was the same as the DefPolicyOffset in the INI.  So, I looked at the 32bit version of the 18362.836 DLL in IDA and found offset 50269 (without cheating!).  Then, I confirmed in the INI that was the same DefPolicyOffset as the 18362.657 DLL.  So I feel like I understand what DefPolicyOffset is, but I don't understand what the others are and I probably wouldn't be able to find them in a different version of the DLL.

LocalOnlyOffset
![image](https://user-images.githubusercontent.com/21339238/82168133-9ff51c00-9883-11ea-9ebe-ad6e34ef9313.png)

SingleUserOffset (I might could find this one)
![image](https://user-images.githubusercontent.com/21339238/82168429-5ce77880-9884-11ea-8d0c-c3100d636df6.png)

SLInitOffset (some kind of api import or export?)
![image](https://user-images.githubusercontent.com/21339238/82168707-10506d00-9885-11ea-81ba-e66b3acfcee6.png)

So what do you think?  Am I trainable to be elite like you where I could maybe help you guys maintain the INI, or should I just accept the fact that i'm a lamer and always will be and I just need to leave the real stuff up to you guys?

asmtron commented:
Hi,

 

thx,

 


add support for termsrv.dll 10.0.18362.836 (x86), you will find the right function with address much easier if you compare the new termsrv.dll with an older working termsrv.dll


 


Sent: Sunday, May 17, 2020 at 6:44 PM
From: "shippj" <notifications@github.com>
To: "asmtron/rdpwrap" <rdpwrap@noreply.github.com>
Cc: "asmtron" <asmtron@mail.com>, "Mention" <mention@noreply.github.com>
Subject: Re: [asmtron/rdpwrap] Updates with new builds (#1)


 

here ya go, thanks!

termsrv.zip

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.

shubell committed:
Update rdpwrap.ini

Compared to my rdpwrap.ini file, added a few comments and a few version. For some I am not sure which is the correct version.

shubell committed:
Added extra entries

Copied some lines from https://github.com/affinityv/INI-RDPWRAP
